### ISSUE TITLE
[Bug]: Sign-Up not appear as link

### ISSUE THREAD

**Issue by KAJAL7764** [Trisha-tech/OnlineBookSales#630]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

The 'Sign-Up' text is not appearing as a clickable link on the registration page. It could be a styled as a link, allowing users to click and navigate to the sign-up form.

### Expected behavior

I expected the 'Sign-Up' text to appear as a clickable link, styled distinctly to indicate that users can click on it to navigate to the registration page."

### Add ScreenShots

![notvisibleaslink](https://github.com/user-attachments/assets/c66afbb9-4076-4324-b4ce-e222cd839557)

Please assign me this task, I want to work on this task.

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

**Comment by JOSHITHA6** [comment]
Yeah even I would love to work on this issue. Please assign me under gssoc'24 extd and hacktoberfest @Trisha-tech

**Comment by shaurya35** [comment]
@KAJAL7764 go ahead with it, assigned to you
